packages/components/transcript-editor/index.js

```diff
@@ -56,6 +46,64 @@ class TranscriptEditor extends React.Component {
     return null;
   }
 
+  // performance optimization
```

Comment:
In this block, we have every single part of the state accounted for and checked, plus also one Prop (mediaUrl).

I checked there weren't any other special cases, but no it's looking at every single state property. So!

Instead of doing all that, we could write:

```
  shouldComponentUpdate = (nextProps, nextState) => {
     if (nextProps.mediaUrl !== this.props.mediaUrl) return true;

     return nextState !== this.state;
  }
```

Reply:
I don't see any difference in performance or side-effects from that. If this is alright, I think we should do the same thing in the other places (as it's adding a lot of verbose and redundant code).